src/ui/helper/ClassSearchFrame.java

```diff
@@ -88,6 +91,14 @@ public boolean isCellEditable(int row, int column) {
 		searchingTextField.getDocument().addDocumentListener(this);
 	}
 
+	/*
+	 * Constructor with a AddLazyJavaClassAction ref to be stored
+	 */
+	public ClassSearchFrame(AddLazyJavaClassAction e) {
```

Comment:
This is the incorrect way of using this. Class Search Frame should not depend on a specific action. Please use the delegates.